aten/src/TH/THAllocator.c

```diff
@@ -82,6 +84,15 @@ THMapAllocatorContext *THMapAllocatorContext_new(const char *filename, int flags
   } else {
     ctx->filename = unknown_filename;
   }
+
+  #ifdef _WIN32
+  char *suffixname = "_event";
+  size_t namelen = strlen(filename)+1+strlen(suffixname);
+  ctx->eventname = THAlloc(namelen);
+  strcpy_s(ctx->eventname, namelen, ctx->filename);
+  strcat_s(ctx->eventname, namelen, suffixname);
```

Comment:
I don't think we want to depend on C11 functions there, especially that you're not really doing any error checking (which is their only benefit compared to older alternatives)